src/Symfony/Bundle/WebServerBundle/Command/ServerLogCommand.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebServerBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\VarDumper\Cloner\Data;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class ServerLogCommand extends Command
+{
+    private $dumper;
+    private $output;
+    private $el;
+
+    private static $levelColorMap = array(
+        100 => 'fg=white',
+        200 => 'fg=green',
+        250 => 'fg=blue',
+        300 => 'fg=cyan',
+        400 => 'fg=yellow',
+        500 => 'fg=red',
+        550 => 'fg=red',
+        600 => 'fg=white;bg=red',
+    );
+
+    private static $bgColor = array(
+        'black',
+        'blue',
+        'cyan',
+        'green',
+        'magenta',
+        'red',
+        'white',
+        'yellow',
+    );
+
+    protected function configure()
+    {
+        $this
+            ->setName('server:log')
+            ->setDescription('Start a log server that displays logs in real time')
+            ->addOption('host', null, InputOption::VALUE_REQUIRED, 'The server host', '0:9911')
+            ->addOption('date-format', null, InputOption::VALUE_REQUIRED, 'The date format', 'H:i:s')
+            ->addOption('format', null, InputOption::VALUE_REQUIRED, 'The line format', '%s <%s>%-9s</> <options=bold>%-8.8s</> %s')
+            ->addOption('show-context', null, InputOption::VALUE_NONE, 'Display the context')
```

Comment:
should we really have options for that ? Couldn't it simply be based on the verbosity instead ? This seems like a perfect use case for it: display more details in verbose mode.